docs/menu.cue

```diff
@@ -745,6 +746,28 @@
 	}]
 }
 
+#SelfHostingSection: #SectionMenuItem & {
+	kind: "section"
+	text: "Self Hosting"
+	items: [
+		{
+			kind: "basic"
+			text: "Build Docker Images"
+			path: "/how-to/self-host"
```

Comment:
Should we make this `/self-host/docker-build` or something? And redirect the old path?